src/app/category-page/category-page.component.ts

```diff
@@ -119,4 +120,18 @@ export class CategoryPage implements OnInit, OnDestroy {
     this.subscriptions.forEach((sub) => { sub.unsubscribe(); });
   }
 
+  public skipToFilterSec(): void{
+    window.setTimeout(() => {
+      let htmlelement = this._dom.byId('skip-to-search-link');
+      htmlelement.focus();
+    }, 100);
+  }
+
+  public skipToSearchSec(): void{
```

Comment:
same thing here

Reply:
@willschoener  updated in the next commit `dd77c08`.